changelog.d/1-api-changes/api-versioning

```diff
@@ -0,0 +1,30 @@
+Start version 2 of the public API. Main changes:
+
+ - Asset endpoints have lost their `v3` and `v4` suffixes. So for example
+   `/assets/v3` has been replaced by `/access`.
```

Comment:
I hope this is a typo?  :)

```suggestion
   `/assets/v3` has been replaced by `/assets`.
```

Also, isn't that something that could have gone into a separate PR?

Reply:
It could've, but I thought it would be useful to start using the new system immediately. It's kind of like adding tests for X in the same PR where you implement X itself. In fact, it is a bit worrying that we are not making significant use of it for the federation API. I've played around with it in a branch, though, by making some random changes to the API, and it seemed to be working reasonably well there, too.